src/toolbar/toolbar-controller.js

```diff
@@ -34,6 +54,21 @@
 
         normalSizeClick() {
             this.window.restore();
+            this.window.resizeTo(1280, 720, 'top-right');
+        }
+
+        summariseClick() {
+            this.summarised = !this.summarised;
+            this.window.summarised = this.summarised;
+            if (this.summarised) {
+                this.window.resizeTo(230, 500, 'top-right');
+            }
+            else if (this.maximised) {
+                this.window.maximize();
+            }
+            else {
+                this.window.resizeTo(1280, 720, 'top-right');
```

Comment:
I'll lump this in here since it's about restoring states. Maybe it's worth adding a `compact: true` or something to the StoreService so it can remember if a window is in compact view between sessions?